tests/integration/fileTests.py

```diff
@@ -342,6 +350,31 @@ def test_check_status(self):
             self.assertEqual(json["created_on"], datetime.utcnow().strftime("%m/%d/%Y"))
             self.assertEqual(json["last_updated"], submission.updated_at.strftime("%Y-%m-%dT%H:%M:%S"))
 
+    def test_get_obligations(self):
+        """Test get obligations route."""
+        postJson = {"submission_id": self.obligations_submission_id}
+
+        response = self.app.post_json("/v1/get_obligations/", postJson, headers={"x-session-id": self.session_id})
+        json = response.json
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(json["total_obligations"], "5000")
+        self.assertEqual(json["total_procurement_obligations"], "2000")
+        self.assertEqual(json["total_assistance_obligations"], "3000")
```

Comment:
I agree with @bsweger -- this'd be much better as a unit test (e.g. of the `obligationStatsForSubmission` function described above). In the current setup, to understand this test, I need to look at `setUpClass`, follow through to `setupObligations` and potentially `insertObligation`. These two tests as unit tests would look like:

```python
def test_obligationStatsForSubmission_nonzero(database):
    submission = SubmissionFactory()
    financials = [
        AwardFinancialFactory(transaction_amount=1000, piid="1234", fain=None, uri=None, 
                              submission=submission),
        AwardFinancialFactory(transaction_amount=1000, piid="1235", fain=None, uri=None, 
                              submission=submission),
        AwardFinancialFactory(transaction_amount=1000, piid=None, fain="1235", uri=None, 
                              submission=submission),
        AwardFinancialFactory(transaction_amount=1000, piid=None, fain=None, uri="1235", 
                              submission=submission),
        AwardFinancialFactory(transaction_amount=1000, piid=None, fain="1234", uri="1235", 
                              submission=submission),
    ]
    database.session.add_all([submission] + financials)
    database.session.commit()
    assert obligationStatsForSubmission(submission.submission_id) == {
        "total_obligations": "5000",
        "total_procurement_obligations": "2000",
        "total_assistance_obligations": "3000"
    }


def test_obligationStatsForSubmission_zero(database):
    submission = SubmissionFactory()
    # no financials in db
    database.session.add(submission)
    database.session.commit()
    assert obligationStatsForSubmission == {
        "total_obligations": "0",
        "total_procurement_obligations": "0",
        "total_assistance_obligations": "0"
    }
```

The benefit to this approach is that there's very little context needed to read the test from start to finish (we really only need to know that the `database` fixture sets up a database and that `Factory` creates populated models).

We still want to exercise the endpoint, though, so maybe this test would be simplified to:

```python
def test_get_obligations(self):
    submission = SubmissionFactory()
    self.session.add(submission)
    self.session.commit()
    response = self.app.post_json(
        "/v1/get_obligations/", 
        {"submission_id": submission.submission_id},
        headers={"x-session-id": self.session_id}
    )
    assert response.status_code == 200
    assert "total_obligations" in response.json
```